tox.ini

```diff
@@ -31,10 +31,11 @@ setenv =
     devdeps,build_docs,online: HOME = {envtmpdir}
     HIDE_PARFIVE_PROGESS = True
     NO_VERIFY_HELIO_SSL = 1
+    devdeps: PIP_EXTRA_INDEX_URL = https://pkgs.dev.azure.com/astropy-project/astropy/_packaging/nightly/pypi/simple/ https://pypi.anaconda.org/scipy-wheels-nightly/simple
 deps =
     # devdeps is intended to be used to install the latest developer version of key dependencies.
-    devdeps: git+https://github.com/astropy/astropy
-    devdeps: git+https://github.com/matplotlib/matplotlib
+    devdeps: astropy>=0.0.dev0
+    devdeps: matplotlib>=0.0.dev0
```

Comment:
That is some magic.

Reply:
Yeah! Found here: https://pip.pypa.io/en/stable/cli/pip_install/#pre-release-versions